README.md

```diff
@@ -57,20 +57,6 @@ Then the Docker images can be built with [Jib](https://github.com/GoogleContaine
 ./gradlew :core:jibDockerBuild
 ```
 
-The `jibDockerBuild` task occasionally gets stuck while pulling the Eclipse Temurin base image. In this case it usually
-helps to clean the project and stop the Gradle Daemon:
-
-```shell
-./gradlew clean
-./gradlew --stop
-```
-
-When building multiple images at once it can also help to disable parallel builds:
-
-```shell
-./gradlew --no-parallel jibDockerBuild
-```
```

Comment:
Could you verify that this issue is gone on a machine were the task previously got stuck without `--no-parallel`?

Reply:
Yes, I was running into the issue before (the ORT server's `gradle.properties` has `org.gradle.parallel = true`), and could not reproduce it anymore afterwards. But of course, I could have just been lucky, and any verification is welcome.

Reply:
I was mainly curious, because I haven't seen this mentioned in any of discussions about this issue that I found, but maybe I just missed it. I will test it on my computer at least before the merge, and if we still see this issue later we can recover those README hints from the Git history.

Reply:
At least on my machine I could not reproduce the issue with this change.